tasks/main.yml

```diff
@@ -1 +1,62 @@
-# Defaults tasks for role sync2git
+---
+# https://github.com/siteshwar/centos-sync-packages/blob/d24dfb53ce26ac3415d57961664f86484a17735b/setup.sh
+- name: "Setup system to sync packages from RHEL to CentOS"
+  hosts: all
+  tasks:
+    - name: "Install krb5-devel python3-devel python3-pip gcc and python-devel packages"
+      yum:
+        name: krb5-devel, python3-devel, python3-pip, gcc, python-devel
+        state: latest
+
+    - name: "Install koji and gitpython pip modules"
+      pip:
+        name: koji, gitpython
+
+    - name: "Install Red Hat certificate"
+      yum:
+        name: http://hdn.corp.redhat.com/rhel7-csb-stage/RPMS/noarch/redhat-internal-cert-install-0.1-15.el7.csb.noarch.rpm
+        state: present
+
+    # TODO: Is there a better way to do this in ansible ?"
```

Comment:
I'd use a template like <name>.repo.j2 and drop it in correct place
See ansible-role-baseline for example